testing/functional_tests_assertions.rst

```diff
@@ -39,6 +39,13 @@ Response
 - ``assertResponseHasCookie()``
 - ``assertResponseNotHasCookie()``
 - ``assertResponseCookieValueSame()``
+- ``assertResponseFormatSame()``, it returns the format associated to the MIME
+  type of the ``Content-Type`` of the response (e.g. it returns ``jsonld`` for
+  ``application/ld+json`` content type).
```

Comment:
These formats are what is returned from `Request::getFormat()`. I would mention that method in this description, as I'm unsure what other formats would be available based on one example.

Reply:
Good idea! Done!